README.md

```diff
@@ -19,12 +19,148 @@ Python packages with our recommended tooling set up and ready to go.
 > package template (e.g., [`scikit-hep`](https://github.com/scikit-hep/cookie)),
 > we recommend using their template instead of this one.
 
+## Tutorial
+
+Some quick instructions for using our template are below.
+We also have a pedagogical [tutorial](./tutorial.md) that has been given in a couple of workshops geared towards researchers at UCL.
```

Comment:
```suggestion
We also have a [tutorial](./tutorial.md) that has been presented in a couple of workshops aimed at researchers at UCL.
```

Reply:
I put in {pedagogical, geared at} because it was in the old sentence that followed.

Don't feel strongly but will leave it for Matt's opinion when he's back.

Reply:
Also don't have strong opinions here so happy for David's suggestion to go in!